lib/rubocop/cop/rspec/excessive_docstring_spacing.rb

```diff
@@ -52,14 +52,14 @@ def on_send(node)
 
         # @param text [String]
         def excessive_whitespace?(text)
-          return true if text.start_with?(' ') || text.end_with?(' ')
+          return true if text.start_with?(' ', '　') || text.end_with?(' ', '　')
 
           text.match?(/[^\n ]  +[^ ]/)
         end
 
         # @param text [String]
         def strip_excessive_whitespace(text)
-          text.strip.gsub(/  +/, ' ')
+          text.strip.gsub(/(\s|　)(\s|　)+/, ' ')
```

Comment:
Like above, let’s not list all different whitespace characters.
```suggestion
          text.strip.gsub(/[[:blank:]]{2,}/, ' ')
```